### PR TITLE
TRAFODION-2873. Cleanup of LobGlobals and LobLoad classes

### PR DESCRIPTION
**Pull request by sandhyasun**
This fix also avoids a core that was very occasionally caused during inserts into LOB tables. The fix involves cleaning up the LOBGlibals and eliminating unneeded datamemebers and keeping only the ExLobGlobals members. Also cleaned up all LobLoad related code since it is obsolete and worked only with DP2.

### CODE REVIEW

File: core/sql/exp/exp_clause.cpp [hunk 1]
```diff
@@ -574,8 +574,8 @@ ex_clause::ex_clause(clause_type type,
 	case ITM_LOBCONVERTHANDLE:
 	  setClassID(LOB_CONVERTHANDLE);
 	  break;
-	case ITM_LOBLOAD:
-	  setClassID(LOB_LOAD);
+          /*	case ITM_LOBLOAD:
```
Comment (DaveBirdsall) [inline core/sql/exp/exp_clause.cpp]: Maybe delete these lines (and the break after them) instead of commenting out?

**Comment by DaveBirdsall** [comment]
Seabase suite build failed for unknown reasons... starting a retest

**Comment by DaveBirdsall** [comment]
jenkins, retest

**Comment by selvaganesang** [comment]
Is ExLobGlobals pertain to the Statement or ExGlobals or the ContextCli or the process?

**Comment by sandhyasun** [comment]
The ExLobGlobals is created whenever a statement or internal API needs to access the LOB functions. It maintains the hdfs connection info, the various lob pointers it needs to access (based on name)  .It is currently a member of ex_globals.It's initialized once per TCB.

**Comment by selvaganesang** [comment]
If so, won't it better to create it using heap_ associated with ex_globals. If there is a connection info that can be shared across context or ESP fragments s, separate them out and tore it in cli_globals and pass that info to ExLobGlobals in ex_globals.  It is strange to see the ExGlobals being constructed from system heap.  And I couldn't find where is it being deleted.  There is a possibility of memory leak.

**Comment by sandhyasun** [comment]
The ExLobGlobals is deleted in  ExpLOBinterfaceCleanup. Will look at getting rid of system new.  There was some issue with using the Map template class with an internal heap. The lobPtrs are stored as a standard template class within the ExLobGlobals.

**Pull request merged**